src/lib/data/alignment.js

```diff
@@ -65,21 +79,41 @@ export default class Alignment {
 
   /**
    * Updates/adds target docSource only if origin is defined
-   * @param {SourceText} docSource
+   * @param {SourceText | Object} docSource
+   * @returns {Boolean}
    */
-  updateTargetDocSource (docSource) {
+  updateTargetDocSource (docSource, targetId) {
     if (!this.origin.docSource) {
-      console.error(L10nSingleton.getMsgS('ALIGNMENT_ERROR_ADD_TO_ALIGNMENT'))
+      if (docSource) {
+        console.error(L10nSingleton.getMsgS('ALIGNMENT_ERROR_ADD_TO_ALIGNMENT'))

```

Comment:
At the start of the application we have an empty text blocks without docSource.
I've added this check to prevent from printing an error in this case (on application init)